.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+indent_size = 4
```

Comment:
Would be nice to switch to 2 spaces and prettier while we're at it, but we can just follow up with that later.